src/test/regress/upgrade/citus_upgrade_test.py

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env python3
+
+"""citus_upgrade_test
+Usage:
+    citus_upgrade_test [options] --bindir=<bindir> --pgxsdir=<pgxsdir>
+
+Options:
+    --bindir=<bindir>                      The PostgreSQL executable directory(ex: '~/.pgenv/pgsql-11.3/bin')
+    --citus-pre-tar=<citus-pre-tar>        Tarball with the citus artifacts to use as the base version to upgrade from
+    --citus-post-tar=<citus-post-tar>      Tarball with the citus artifacts to use as the new version to upgrade to
+    --pgxsdir=<pgxsdir>           	       Path to the PGXS directory(ex: ~/.pgenv/src/postgresql-11.3)
+    --mixed                                Run the verification phase with one node not upgraded.
+"""
+
+import subprocess
+import atexit
+import os
+import re
+import sys
+
+import utils
+
+from docopt import docopt
+
+from config import (
+    CitusUpgradeConfig, NODE_PORTS, COORDINATOR_NAME, CITUS_VERSION_SQL, MASTER_VERSION,
+    NODE_NAMES, USER, WORKER1PORT, MASTER, HOME,
+    AFTER_CITUS_UPGRADE_COORD_SCHEDULE, BEFORE_CITUS_UPGRADE_COORD_SCHEDULE
+)
+from upgrade_common import initialize_temp_dir, initialize_citus_cluster, run_pg_regress, stop_databases
```

Comment:
maybe import as common and use `common.<name>` in the code. Makes it slightly easier to follow while reading where the code is located when called.